jobs/insert_new_games.py

```diff
@@ -1,7 +1,21 @@
 import utils 
 
 if __name__ == '__main__':
+
+    '''DEPRECATED
     scrap = utils.Scrap()
     df_raw = scrap.get_game_schedule_dataframe()
     df_clean =scrap.clean_schedule(df_raw)
-    scrap.insert_future_games(df_clean)
\ No newline at end of file
+    scrap.insert_future_games(df_clean)
+    '''
+    
+    psr = utils.pandaScoreRequest()
```

Comment:
En l'état génère une erreur lors du lancement de insert_new_games.py
```suggestion
    psr = utils.PandaScoreRequest()
```